extensions/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java

```diff
@@ -585,6 +585,9 @@ private static void checkParameterNames(IndexView index,
         OUTER: for (DotName annotationType : methodParameterAnnotations) {
             Collection<AnnotationInstance> instances = index.getAnnotations(annotationType);
             for (AnnotationInstance instance : instances) {
+                // we only care about method parameters, because properties or fields always work
+                if (instance.target().kind() != Kind.METHOD_PARAMETER)
+                    continue;
```

Comment:
We try to have curly braces even for one line.

Reply:
Indeed, that one slipped they eye. Actually we should probably have a formatting rule for that

Reply:
I force pushed a change for that, and a rebase.